src/vivarium_gates_nutrition_optimization/data/loader.py

```diff
@@ -278,6 +289,32 @@ def load_pregnant_maternal_hemorrhage_incidence(key: str, location: str):
     return maternal_hemorrhage_incidence.applymap(lambda value: 1 if value > 1 else value)
 
 
+###########################
+# Hemoglobin Data         #
+###########################
+
+
+def get_hemoglobin_data(key: str, location: str) -> pd.DataFrame:
+    me_id = {
+        data_keys.HEMOGLOBIN.MEAN: 10487,
+        data_keys.HEMOGLOBIN.STANDARD_DEVIATION: 10488,
+    }[key]
+
+    location_id = utility_data.get_location_id(location)
+    hemoglobin_data = gbd.get_modelable_entity_draws(me_id=me_id, location_id=location_id)
+    hemoglobin_data = reshape_to_vivarium_format(hemoglobin_data, location)
+    # Add correction factors for pregnancies
+    correction_params = data_values.PREGNANCY_CORRECTION_FACTORS[key]
+    dist = sampling.get_norm_from_quantiles(*correction_params)
+    correction_factors = pd.DataFrame(
+        np.tile(dist.rvs(size=1000), (len(hemoglobin_data), 1)),
```

Comment:
do i need to do something special as far as a random seed goes for this?

Reply:
That's a good question - it does seem like randomness needs to play a part somewhere but I'm not even seeing in the stats docs (or np.tile docs) where randomness seed would go. @rmudambi ?

Reply:
the original function does `np.random.seed(get_hash(f'{key}_{sample_status}_{location}'))`  after loading the correction parameters FWIW

Reply:
It's not critical that we set a seed here for reproducibility of our final results, since we'll ultimately ship the artifact with our repository and this call only happens when building that artifact, but it is highly desirable, because that means that every time we rebuild our artifact we should get the same values, and we will likely rebuild it a few times during the course of model development. 

You could just set a random seed as was done in IV Iron, but this functionality has actually been deprecated in `numpy` in favor of using `RandomGenerator` https://numpy.org/doc/stable/reference/random/generator.html

Reply:
@rmudambi @stevebachmeier I updated this with a `RandomGenerator` and verified that the same numbers persist each call/ interpreter session